text/intro.tex

```diff
@@ -1,10 +1,37 @@
 \chapter{Introduction}
 \label{sec:intro}
 
-This document recommends a methodological approach for measuring, recording, and reporting the power used by a high performance computer (HPC) system while running a workload.
+This document recommends a methodological approach for measuring, recording, and reporting the power used 
+by a high performance computer (HPC) system while running a workload.
+
+This document is part of a collaborative effort between the Green500, the Top500, the Green Grid, and the 
+Energy Efficient High Performance Computing Working Group (EEHPC WG). While it is intended for this 
+methodology to be generally applicable to benchmarking a variety of workloads, the initial focus is on 
+High Performance LINPACK (HPL).
+
+Benchmarking computer systems is a way of comparing performance between architectures. There are 
+benchmarks that are used for comparing the energy efficiency of architectures and include both a 
+measure of the benchmark performance and either energy or power. Benchmarking efforts, like the 
+Top500 and the Green500, provide a ranked list of general purpose systems that are in common use 
+for high end applications. They also provide a reliable basis for tracking and detecting trends 
+in high performance computing.
+
+Besides providing more accurate and precise information for the Green500 List, there are other 
+reasons to make a system-level power measurement. There are a lot of capabilities today for 
+measuring component-level power. This is especially true for processors and accelerators, but 
+it also includes node and rack-level measurements. System-level power is still a challenge for 
+many sites and systems. Sites and vendors may want to do architectural trending and system 
+modeling for design, selection, upgrades, tuning and analysis. Another example is the use for 
+understanding the actual --- versus peak --- power draw of an HPC system based on a normal 
+site workload vs. wall-plate power. This might be useful for provisioning the datacenter as 
+there can be a substantial difference between peak and normal power draw. Operation 
+improvements, such as for cooling systems, can leverage the ability to measure HPC system-level 
+power and energy. Finally, system-level measurements can be used to validate component-level 
+measurements by summing them up.
```

Comment:
Please reshuffle the text so that we have _one line per sentence_, to improve diff for any further changes to the text.

Reply:
New push to reformat the paragraphs to have line length at most 80 characters long (hard-set wordwrap limit length).